docs/cpp/config-linux.md

```diff
@@ -109,17 +109,23 @@ In the `helloworld.cpp` file, hover over `vector` or `string` to see type inform
 
 You can press the `kbstyle(Tab)` key to insert the selected member. Then, when you add the opening parenthesis, you'll see information about arguments that the function requires.
 
-## Build helloworld.cpp
+## Run helloworld.cpp
+Remember, the C++ extension uses the C++ compiler you have installed on your machine to build your program. Make sure you have a C++ compiler installed before attempting to run and debug `helloworld.cpp` in VS Code.
 
-Next, you'll create a `tasks.json` file to tell VS Code how to build (compile) the program. This task will invoke the g++ compiler to create an executable file from the source code.
+1. Open `helloworld.cpp` so that it is the active file.
+2. Press the play button in the top right corner of the editor.
 
-It's important to have `helloworld.cpp` open in the editor because the next step uses the active file in the editor for context to create the build task in the next step.
+   ![Screenshot of helloworld.cpp and play button](images/playbutton/run-play-button.png)
+
+3. Choose **g++ build and debug active file** from the list of detected compilers on your system (you'll only be asked to choose a compiler the first time you run `helloworld.cpp`).
```

Comment:
Same as the previous comment:
as a side note: it is **g++ build and debug active file**, not **g++ build active file**,